_posts/2020-12-24-newpipe-0.20.7.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "NewPipe 0.20.7 released: YouTube Mix support, bug fixes and many improvements"
+short: "NewPipe 0.20.7 released"
+date: 2020-12-24 19:00:00
+categories: [pinned, release]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Merry christmas, if you celebrate that, and almost happy new year, if you use a calender at least close to the gregorian one. We want to present you a little gift as a good way to start into 2021 AD: The next release of NewPipe. It comes with a lot of improvement, fixes, and even a big new feature. So let us dive into this release post
```

Comment:
```suggestion
Merry Christmas, if you celebrate that, and almost happy new year, if you use a calendar at least close to the Gregorian one. We want to present you a little gift as a good way to start into 2021 AD: The next release of NewPipe. It comes with a lot of improvements, fixes, and even a big new feature. So let us dive into this release post!
```

Reply:
looks good, we need to check if we make this a new years release or in between the days and change the first sentence accordingly